reviveme/scripts/add_fake_data.py

```diff
@@ -2,23 +2,65 @@
 from reviveme.extensions import db
 from reviveme.models import Comment, Thread, User
 
-app = create_app()
-# TODO: create a migration instead of using create_all for this
-with app.app_context():
-    # Create or update the database schema
-    db.create_all()
-
-    # Add data to the User model
-    user1 = User(
-        username="john_doe", email="john@example.com", password="123john", salt="test"
-    )
-    user2 = User(
-        username="jane_doe", email="jane@example.com", password="123john", salt="test"
-    )
-
-    db.session.add(user1)
-    db.session.add(user2)
-
-    db.session.commit()
-
-    print("Data added successfully.")
+FAKE_USER_DATA = (
+    {
+        "id": 1,
+        "username": "john_doe",
+        "email": "john@example.com",
+        "password": "123john",
+        "salt": "test",
+    },
+    {
+        "id": 2,
+        "username": "jane_doe",
+        "email": "jane@example.com",
+        "password": "123john",
+        "salt": "test",
+    },
+)
+
+FAKE_THREAD_DATA = (
+    {
+        "id": 3,
+        "author_id": 1,
+        "title": "Johns Thread",
+        "content": "Lorem Ipsum...",
+    },
+)
+
+FAKE_COMMENT_DATA = (
+    {
+        "id": 4,
+        "author_id": 2,
+        "thread_id": 3,
+        "content": "This is a comment made by Jane",
+    },
+)
+
+
+def create_fake_database_data():
+    app = create_app()
+    with app.app_context():
+        # Create or update the database schema
+        db.drop_all()
```

Comment:
Would there be any way to run migrations here instead? That way we wouldn't need to blow up our existing data to load the sample data.

Reply:
Yeah this is probably a better idea